korman/properties/modifiers/gui.py

```diff
@@ -330,3 +333,319 @@ def _create_moul_nodes(self, clickable_object, nodes, journalnode, age_name):
+        "filename": "xSimpleLinkingBook.py",
+        "attribs": (
+            { 'id':  1, 'type': "ptAttribActivator", "name": "bookClickable" },
+            { 'id':  2, 'type': "ptAttribString",    "name": "destinationAge" },
+            { 'id':  3, 'type': "ptAttribString",    "name": "spawnPoint" },
+            { 'id':  4, 'type': "ptAttribString",    "name": "linkPanel" },
+            { 'id':  5, 'type': "ptAttribString",    "name": "bookCover" },
+            { 'id':  6, 'type': "ptAttribString",    "name": "stampTexture" },
+            { 'id':  7, 'type': "ptAttribFloat",     "name": "stampX" },
+            { 'id':  8, 'type': "ptAttribFloat",     "name": "stampY" },
+            { 'id':  9, 'type': "ptAttribFloat",     "name": "bookWidth" },
+            { 'id': 10, 'type': "ptAttribFloat",     "name": "BookHeight" },
+            { 'id': 11, 'type': "ptAttribBehavior",  "name": "msbSeekBeforeUI" },
+            { 'id': 12, 'type': "ptAttribResponder", "name": "respOneShot" },
+        )
+    },
+    pvMoul : {
+        "filename": "xLinkingBookGUIPopup.py",
+        "attribs": (
+            { 'id':  1, 'type': "ptAttribActivator", 'name': "actClickableBook" },
+            { 'id':  2, 'type': "ptAttribBehavior",  'name': "SeekBehavior" },
+            { 'id':  3, 'type': "ptAttribResponder", 'name': "respLinkResponder" },
+            { 'id':  4, 'type': "ptAttribString",    'name': "TargetAge" },
+            { 'id':  5, 'type': "ptAttribActivator", 'name': "actBookshelf" },
+            { 'id':  6, 'type': "ptAttribActivator", 'name': "shareRegion" },
+            { 'id':  7, 'type': "ptAttribBehavior",  'name': "shareBookSeek" },
+            { 'id': 10, 'type': "ptAttribBoolean",   'name': "IsDRCStamped" },
+            { 'id': 11, 'type': "ptAttribBoolean",   'name': "ForceThirdPerson" },
+        )
+    },
+}
+
+
+class PlasmaLinkingBookModifier(PlasmaModifierProperties, PlasmaModifierLogicWiz):
+    pl_id = "linkingbookmod"
+
+    bl_category = "GUI"
+    bl_label = "Linking Book"
+    bl_description = "Linking Book"
+    bl_icon = "FILE_IMAGE"
+
+    versions = EnumProperty(name="Export Targets",
+                            description="Plasma versions for which this journal exports",
+                            items=game_versions,
+                            options={"ENUM_FLAG"},
+                            default={"pvMoul"})
+
+    # Link Info
+    link_type = EnumProperty(name="Linking Type",
+                             description="The type of Link this Linking Book will use",
+                             items=[
+                                 ("kBasicLink", "Public Link", "Links to a public instance of the specified Age"),
+                                 ("kOriginalBook", "Private Link", "Links to a new or existing private instance of the specified Age"),
+                                 ("kSubAge", "Closed Loop Link", "Links between instances of the specifed Age and the current one"),],
```

Comment:
`kSubAgeBook`

Reply:
I suspect this was a victim of our changes from explicit to simplified options.